velox/docs/functions/spark/config.rst

```diff
@@ -0,0 +1,21 @@
+================================
+SparkRegistration Configuration
+================================
+
+struct SparkRegistrationConfig
+---------------------
+.. list-table::
+   :widths: 20 10 10 70
+   :header-rows: 1
+
+   * - Property Name
+     - Type
+     - Default Value
+     - Description
+   * - allowPrecisionLoss
+     - bool
+     - true
+     - When true, establishing the result type of an arithmetic operation according to Hive behavior and SQL ANSI 2011 specification, i.e.
+       rounding the decimal part of the result if an exact representation is not
+       possible. Otherwise, NULL is returned when the actual result cannot be represented with the calculated decimal type. Now we support add,
```

Comment:
> NULL is returned

Do we expect NULL output or an exception when the exact representation is not possible?

Reply:
Yes.
AllowPrecisionLoss mode also returns null.
```
spark-sql (default)> select cast("99999999999999999999999999999999999" as decimal(38, 4)) *  cast("999999999999999999999999999999999999" as decimal(38, 4));
NULL
Time taken: 2.298 seconds, Fetched 1 row(s)
```